docs/how_to_add_packages.md

```diff
@@ -144,9 +153,18 @@ Then in your `conanfile.py` method, it has to be used to download the sources:
 ```
 
   The hook will show error messages but the `conan create` won’t fail unless you export the environment variable `CONAN_HOOK_ERROR_LEVEL=40`.
+  All hook checks will print a similar message:
 
-Call `conan create . lib/1.0@` in the folder of the recipe using the profile you want to test.
+  [HOOK - conan-center.py] post_source(): [LIBCXX MANAGEMENT (KB-H011)] OK
+  [HOOK - conan-center.py] post_package(): ERROR: [PACKAGE LICENSE] No package licenses found
+
+Call `conan create . lib/1.0@` in the folder of the recipe using the profile you want to test. For instance:
+
+    cd conan-center-index/recipes/boost/all
+    conan create . 1.74.0@
 
 ### Debugging failed builds
 
-   Go to the [Error Knowledge Base](error_knowledge_base.md) page to know more.
+   Go to the [Error Knowledge Base](error_knowledge_base.md) page to know more about Conan Center hook errors.
+
+   Some common errors related to Conan can be found on [troubleshooting](https://docs.conan.io/en/latest/faq/troubleshooting.html) section.
```

Comment:
I would like to provide something more useful. Debugging broken recipes is not an easy task, and for newbies is worst, probably they will give up before pushing the PR. Maybe, what should I check when packaging? Compare the license file with SDPX website? Read the original CMakeLists.txt?

Reply:
From the few I've done, upstream CMake targets requiring patching... minimum compiler support is very common.... it took me a while to figure out trying non-default option values (shared imports on windows)

Reply:
Well, there are so many things to check:
- PIC forced: search for pattern like `POSITION_INDEPENDANT_CODE`, `-fPIC`
- msvc runtime overriden
- broken installation folders (usually runtime installed to lib folder instead of bin)
- static and shared built together (eventually through a common OBJECT target, breaking non PIC or preventing proper export of symbols with msvc).
- system libs used instead of conan libs
- how to analyse a CMakeLists to deduce config file name and imported target names
- variables like CMAKE_SOURCE_DIR used instead of CMAKE_CURRENT_SOURCE_DIR
- RPATH hardcoded in shared libs
- static dependencies merged together (ODR violation)
- symbols not exported with msvc

Reply:
Maybe we could create another doc, like "points to be reviewed", a document more detailed in terms of recipe construction, otherwise this document will get bigger and bigger